github-profiles/Marcos.-0222/style.css

```diff
@@ -0,0 +1,106 @@
+@import 'https://fonts.googleapis.com/css?family=Roboto:300,400,500';
+
+header a {
```

Comment:
You do not have a header tag on your `.html` file, so you can delete styles for header or even `header a` on your `.css` file